### DISCUSSION TITLE
Missing files, tool map and work map

### DISCUSSION THREAD

**Discussion by josephs07** [freeplane/docs#131] [Q&A] [answered]
I am trying to get better at problem solving with freeplane. Does anyone have the files mentioned in this page
[step-by-step-instructions](https://docs.freeplane.org/attic/old-mediawiki-content/Freeplane_and_problem_solving.html#step-by-step-instructions)

**Comment by macmarrum** [comment, marked as answer]
The page is in `attic`. It was moved/migrated from the old MediaWiki site but only the Markdown/text content, without attachments.

The old site is still available. You can search for the page there: https://www.freeplane.org/oldwiki

**Reply by josephs07** [reply]
found them here https://www.freeplane.org/oldwiki/index.php/Freeplane_and_problem_solving
Thanks
